pkg/health/client/modules.go

```diff
@@ -26,18 +26,22 @@ func GetAndFormatModulesHealth(w io.Writer, clt ModulesHealth, verbose bool) {
 		fmt.Fprintf(w, "\t%s\n", err)
 		return
 	}
+
 	if resp.Payload == nil {
 		fmt.Fprintf(w, "\tno health payload detected\n")
 		return
 	}
 	if verbose {
 		fmt.Fprintf(w, "\n  Module\tStatus\tMessage\tLast Updated\n")
 		for _, m := range resp.Payload.Modules {
-			if strings.Contains(m.Message, "\n") {
-				fmt.Fprintf(w, "  %s\t%s\t%12s\n%s", m.ModuleID, m.Level, m.LastUpdated, m.Message)
-			} else {
-				fmt.Fprintf(w, "  %s\t%s\t%s\t%12s\n", m.ModuleID, m.Level, m.Message, m.LastUpdated)
+			n := &cell.StatusNode{}
```

Comment:
**Note**: This is a temporary measure - following this PR we will be able to change the OpenAPI model to pass the [entire update structure](https://github.com/cilium/cilium/pull/28804/files#diff-508a09ed0e66f45b2f1c78e3650eedf4de551b63150c6dac457298af46f9152bR1388) instead of sending the data as a json string.

Reply:
I am actually not sure about this? Thinking this data will eventually be stored in state db. Likely this would results in a collection of rows when queried and it will be up to the call site to render a tree if needed?

Reply:
@joamaki this makes sense, as mentioned I have a follow up to this PR where I was going to change the openapi spec to include the tree structure but I think i'll change that to have a flat list of nodes/id/parent-ids to be close to an eventual statedb schema.

Reply:
Re: discussion points with @derailed regarding statedb, discussing this offline we've compromised to continue cleaning up current iteration of modular health reporting and begin looking at possibly transitioning to statedb following the next release.

Will resolve